ada_2_fhir-r4/cio/2.0.0/beschikbaarstellen_overgevoeligheid/fhir_instance/ov-Ov-Groep-HEPARINEGROEP.xml

```diff
@@ -0,0 +1,86 @@
+<Condition xmlns="http://hl7.org/fhir">
+   <id value="ov-Ov-Groep-HEPARINEGROEP"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/cio-HypersensitivityDisposition"/>
+   </meta>
+   <extension url="http://hl7.org/fhir/StructureDefinition/condition-dueTo">
+      <valueCodeableConcept>
+         <coding>
+            <system value="urn:oid:2.16.840.1.113883.2.4.4.1.902.122"/>
+            <code value="23"/>
+            <display value="HEPARINEGROEP"/>
+         </coding>
+      </valueCodeableConcept>
+   </extension>
+   <extension url="http://hl7.org/fhir/StructureDefinition/condition-assertedDate">
+      <valueDateTime value="${DATE, T, D, -270}T09:30:22+02:00"/>
+   </extension>
+   <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-HypersensitivityDisposition.MethodOfEstablishment">
+      <valueCodeableConcept>
+         <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="144511000146101"/>
+            <display value="overgevoeligheid vastgesteld op basis van klinisch beeld en aanvullend onderzoek (bevinding)"/>
+         </coding>
+      </valueCodeableConcept>
+   </extension>
+   <identifier>
+      <system value="urn:oid:2.16.840.1.113883.2.4.3.11.999.26.1.3"/>
+      <value value="Ov-Groep HEPARINEGROEP"/>
+   </identifier>
+   <clinicalStatus>
+      <coding>
+         <system value="http://terminology.hl7.org/CodeSystem/condition-clinical"/>
+         <code value="active"/>
+         <display value="Active"/>
+      </coding>
+   </clinicalStatus>
+   <category>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="419511003"/>
+         <display value="geneesmiddelen"/>
+      </coding>
+   </category>
+   <code>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="157471000146100"/>
+         <display value="overgevoeligheid aanwezig"/>
+      </coding>
+   </code>
+   <subject>
+      <reference value="Patient/patient-XXX-Gerrits"/>
+      <type value="Patient"/>
+      <display value="Patient, O. XXX_Gerrits"/>
+   </subject>
+   <onsetDateTime value="${DATE, T, D, -274}"/>
+   <recordedDate value="${DATE, T, D, -270}T09:30:22+02:00"/>
+   <recorder>
+      <reference value="PractitionerRole/pracrole-000003337"/>
+      <type value="PractitionerRole"/>
+      <display value="Healthcare professional (role), A. van Groenekruis, Apothekers, Algemeen Stadsziekenhuis"/>
+   </recorder>
+   <asserter>
+      <reference value="PractitionerRole/pracrole-000003337"/>
+      <type value="PractitionerRole"/>
+      <display value="Healthcare professional (role), A. van Groenekruis, Apothekers, Algemeen Stadsziekenhuis"/>
+   </asserter>
+   <evidence>
+      <code>
+         <extension url="http://hl7.org/fhir/StructureDefinition/iso21090-nullFlavor">
+            <valueCode value="OTH"/>
+         </extension>
+         <text value="trombocytopenie"/>
```

Comment:
Is mij niet helemaal duidelijk dat deze waarde hier wordt verwacht

Reply:
Deze waarde komt overeen met 'Reactie verschijnsel' uit de Reactie die gekoppeld is aan de Overgevoeligheid. Is de toelichting op https://simplifier.net/test-cio/ciohypersensitivitydisposition, `Condition.evidence.code` wat jou betreft voldoende?

Reply:
ah duidelijk! misschien vermelden in kwalificatiemateriaal?

Reply:
Als het profiel op Condition blijft bestaan, eventueel verder verduidelijken (kan ook in TO), maar voor nu geen verdere aanpassing. Aanpassing in kwalificatiemateriaal is lastig vanwege het automatisch genereren m.b.v. ada2wiki.